### PR TITLE
Pull request for new example "diffusion_equation_with_linear_source"

### PR DESCRIPTION
**Pull request by kmith**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+   
+Building the example
+====================
+
+The fortran version of the example can be configured and built with CMake::
+
+  git clone https://github.com/OpenCMISS-Examples/diffusion_equation_with_linear_source
+  mkdir diffusion_equation_with_linear_source-build
+  cd diffusion_equation_with_linear_source-build
+  cmake -DOpenCMISSLibs_DIR=/path/to/opencmisslib/install ../diffusion_equation_with_linear_source
```
Comment (hsorby) [inline README.rst:61]: -DOpenCMISSLibs_DIR=/path/to/opencmisslibs/install

Added an 's'.

**Review by hsorby — CHANGES_REQUESTED** [review]
I don't see a Python script file that you mention in the README.

File: README.rst (hunk 1 above)
Comment (kmith) [reply README.rst:61]: This example doesn't have a python script. The four examples I have committed so far were existing examples which I modified conforming to the new format. Let me know if you want me get rid of the statement on python version. I hope we can include python version sometime in the future. Also note that the previous example "diffusion_equation" also did not have the python script.

**Comment by hsorby** [comment]
If the example doesn't have a Python script then the README shouldn't mention how to run the Python version.  If there is no Python script then yes remove any mention of Python from the README (all README's).

**Comment by kmith** [comment]
Removed python related info as this example does not have a python script.

**Review by hsorby — APPROVED** [review]
That's better it now matches what is actually there.

**Pull request merged**